gem/views.py

```diff
@@ -280,6 +281,32 @@ def get(self, request, comment_pk):
         })
 
 
+class KaiOSManifestView(View):
+    def get(self, request):
+        manifest = {
+            "version": "1.0.0",
+            "name": "Springster App",
+            "description": "An app providing information to girls",
+            "launch_path": "/",
+            "icons": {
+                "56": "/gem/statc/img/icons/next.png",
```

Comment:
This well probably be something like `/static/img/icons/next.png`

Reply:
ok